smart-contracts/Cargo.toml

```diff
@@ -17,6 +17,7 @@ members = [
   "contracts/dex-router-osmosis",
   "contracts/lst-dex-adapter-osmosis",
   "contracts/lst-adapter-osmosis",
+  "contracts/babylon-vault",
 ]
 
 [workspace.dependencies]
```

Comment:
I dont think it makes sense to use the same workspace as osmosis-contracts as the target chain isn't osmosis

Reply:
We had discussed that when starting with the monorepo (one workspace, different suffixes). If there is not a very strong reason to do so, I would prefer to not open that bottle again.

Reply:
this worksapce has osmosis dependencies, i think we decided product based worksapces cause target chains are different

Reply:
Makes sense to change it later once we have more segregations in chains. Right now everything in same workspace is okay.

But in general if we are prepping to move towards mono-repo (multiple destination contracts) discussion we had then shouldn't we start it now? Might be a first step in right direction.

Reply:
Will create two separate workspaces osmosis and babylon and then continue on this PR.

Reply:
done